packages/plugin-essentials/sources/commands/install.ts

```diff
@@ -26,6 +26,10 @@ export default class YarnCommand extends BaseCommand {
 
       - **Build:** Once the dependency tree has been written on the disk, the package manager will now be free to run the build scripts for all packages that might need it, in a topological order compatible with the way they depend on one another. See https://yarnpkg.com/advanced/lifecycle-scripts for detail.
 
+      Packages present in the `node_modules` folder but not in the `package.json` will be automatically pruned.
+      
+      If you are using workspaces, the installation process will respect their topology and create appropriate symlinks.
```

Comment:
This is specific to the `node-modules` linker, not installs in general. We should instead reword `This command sets up your project if needed` to indicate that the linker **_might_** clean up.

We should also remove the `Note that running this command is not part of the recommended workflow` section as it was written a long time ago and isn't accurate anymore.